xarray/tests/test_sparse.py

```diff
@@ -873,3 +873,16 @@ def test_dask_token():
     t5 = dask.base.tokenize(ac + 1)
     assert t4 != t5
     assert isinstance(ac.data._meta, sparse.COO)
+
+
+@requires_dask
```

Comment:
Should this also have `@requires_sparse`?

Reply:
There's a `importorskip("sparse")` on top so I think this is fine